p2p/pubsubManager/gossipsub.go

```diff
@@ -36,7 +36,10 @@ type PubsubManager struct {
 // TODO: what options do we need for quai network? See:
 // See https://pkg.go.dev/github.com/libp2p/go-libp2p-pubsub@v0.10.0#Option
 func NewGossipSubManager(ctx context.Context, h host.Host) (*PubsubManager, error) {
-	ps, err := pubsub.NewGossipSub(ctx, h)
+	cfg := pubsub.DefaultGossipSubParams()
+	cfg.D = 10
+	cfg.Dout = 5
```

Comment:
Please use C_peerCount from stream manager and set it to 5 instead of specifying a raw number here. Or I can add a commit on top to accomplish this, let me know